build/BuildEnv.shade

```diff
@@ -3,6 +3,26 @@ use namespace="System"
 
 functions
   @{
+    string FileVersion 
+    {
+        get
+        {
+            DateTime start = new DateTime(2015, 1, 1);
+            DateTime now = DateTime.UtcNow;
+            
+            string version = "0";
+
+            // If the computer date is set before the start date, then the version is 0
+            if (now >= start)
+            {
+                int monthsSinceStart = (now.Year - start.Year) * 12 + now.Month;
+                version = monthsSinceStart.ToString("000") + now.Day.ToString("00");
+            }
+
+            return "1.0." + version + ".0";
```

Comment:
This needs to match the `AssemblyVersion`'s major and minor version. So Mvc is `6.0.` something.